_summaries/wu2021rethinking_1.md

```diff
@@ -1,19 +1,180 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
+title: Rethinking “Batch” in BatchNorm
+paper: wu2021rethinking_1
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: # chengchunhsu
+score: 8/10
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+**A short review on BatchNorm**
+
+BatchNorm is sometimes viewed as a “necessary evil” in the design of CNNs
+
+- It improves model convergence speed
+
+- regularizing effect to combat overﬁtting
+
+
+
+![BN_EQ1](wu2021rethinking_1_1.jpg)
+
+![BN EQ2](wu2021rethinking_2.jpg)
```

Comment:
I wonder if possible to change the formula to latex, it looks better than the image,

$$y=\frac{x-\mu}{\sqrt{\sigma^2+\epsilon}}$$ for the first image?